sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -121,3 +121,12 @@ export const DUPLICATE_INSTITUTION_LOCATION_CODE =
  * Duplicate SABC code for the institution.
  */
 export const DUPLICATE_SABC_CODE = "DUPLICATE_SABC_CODE";
+
+export const COE_NOT_FOUND_MESSAGE =
```

Comment:
this is an error message, right? but the file name is `error-code.constants`.

Reply:
Most of the time I would say that the error names are constants and the error messages aren't, with few exceptions like these ones. I am not sure what was the motivation to have these ones as constants in the past.